samples/dect/dect_phy/dect_shell/src/dect/dect_phy_shell.c

```diff
@@ -889,6 +901,28 @@ static int dect_phy_ping_cmd(const struct shell *shell, size_t argc, char **argv
 			params.tx_mcs = atoi(optarg);
 			break;
 		}
+		case DECT_SHELL_PING_TX_LBT_PERIOD: {
+			tmp_value = atoi(optarg);
+			if (tmp_value != 0 && (tmp_value < DECT_PHY_LBT_PERIOD_MIN_SYM ||
+					       tmp_value > DECT_PHY_LBT_PERIOD_MAX_SYM)) {
+				desh_error("Give decent LBT period (range: [%d,%d])",
+					   DECT_PHY_LBT_PERIOD_MIN_SYM,
+					   DECT_PHY_LBT_PERIOD_MAX_SYM);
```

Comment:
I would mention the given value that is invalid. So something like:
```suggestion
				desh_error("LBT period (%d) out of range [%d,%d])",
					   tmp_value,
					   DECT_PHY_LBT_PERIOD_MIN_SYM,
					   DECT_PHY_LBT_PERIOD_MAX_SYM);
```

If you do this, do similar change to `DECT_SHELL_PING_TX_LBT_RSSI_BUSY_THRESHOLD`.